src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx

```diff
@@ -126,38 +122,32 @@ export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkForm
             >
               {query}
             </Text>
-            “ on Artsy.{" "}
-            {!!enablePersonalArtists &&
-              "You can add their name in the artwork details."}
+            “ on Artsy. "You can add their name in the artwork details."
```

Comment:
```suggestion
            “ on Artsy. You can add their name in the artwork details."
```

Reply:
thanks- I will fix that

Reply:
Let's check this on staging when it's deployed. I'm still quite confused by all the `"`'s :)